### PR TITLE
Fix two typos in sort_points

### PR DESCRIPTION
**Pull request by jwboth**
Fix two small typos.

### CODE REVIEW

File: src/porepy/utils/sort_points.py [hunk 1]
```diff
@@ -22,21 +21,21 @@ def sort_point_pairs(
...
-    surely be imporved.
+    surely be improved.
 
     Parameters:
     lines: np.ndarray, 2xn, the line pairs. If lines has more than 2 rows, we assume
         that the points are stored in the first two rows.
     check_circular: Verify that the sorted polyline form a circle.
-                    Defaluts to true.
+                    Defaults to true.
     ordering: np.array, return in the original order if a line is flipped or not
```
Comment (IvarStefansson) [inline src/porepy/utils/sort_points.py:31]: The description of "ordering" is not intuitive to me. Could you improve it if you understand it?

**Review by IvarStefansson — APPROVED** [review]

**Review by keileg — APPROVED** [review]
Thanks. As indicated, I believe @IvarStefansson's question about the ordering parameter can be solved by deletion of the relevant code. Could you please have a quick look at this before merging?

File: src/porepy/utils/sort_points.py (hunk 1 above)
Comment (jwboth) [reply src/porepy/utils/sort_points.py:31]: The parameter `ordering` has still an effect. If set `True`, a boolean array is returned specifying which of the columns in `lines` had to be flipped in order to construct a chain. If we want to keep this option, I suggest changing the name to something like `return_flip_info` as well as changing the corresponding boolean array `is_ordered` to something like `is_flipped`. On the other hand, this option is actually nowhere used in PP core. 

Any opinion on whether to keep this option? If we want to remove it, we can also remove anything related to `is_ordered`.

Comment (keileg) [reply src/porepy/utils/sort_points.py:31]: I think I know where it is used (in an external package of mine). Let's kill it, then I will deal with the consequences.

**Pull request merged**